tests/drivers/build_all/sensor/i2c.dtsi

```diff
@@ -56,7 +56,14 @@ test_i2c_apds9960: apds9960@7 {
 	int-gpios = <&test_gpio 0 0>;
 };
 
-test_i2c_bma280: bma280@8 {
+test_i2c_bh1750: bh1750@23 {
+	compatible = "rohm,bh1750";
+	reg = <0x23>;
+	mtreg = <69>;
+	resolution = <1>;
+};
```

Comment:
please see the comment at the top of the file about keeping addresses sequential